### PR TITLE
Multiple sequences can be sent in parallel manner

### PR DESCRIPTION
**Pull request by santosh-kavhar**
Multiple sequences can be sent in parallel manner with new sequence showing only a warning. Earlier it used to show error and not process the new sequence.

### CODE REVIEW

File: lib/python/satos_payload_sdk/app_framework.py [hunk 1]
```diff
@@ -371,8 +371,7 @@ def start_sequence(self, seq_id, seq_params, seq_deadline):
 
         with self.lock:
```
Comment (deepaktawri) [inline lib/python/satos_payload_sdk/app_framework.py:372]: We should remove the lock related code altogether.

**Review by deepaktawri — CHANGES_REQUESTED** [review]

File: lib/python/satos_payload_sdk/app_framework.py (hunk 1 above)
Comment (rahulbhivare) [reply lib/python/satos_payload_sdk/app_framework.py:372]: One question : If a specific sequence is in progress, then are we allowing user to call same sequence again ? Or we want a scheme where user can call multiple sequences but can not call same sequence again ? Considering design expectation of "Sending a single sequence at the start of AOI and one at the end of AOI", I believe we should bypass different sequence and do not send same sequence.

Comment (deepaktawri) [reply lib/python/satos_payload_sdk/app_framework.py:372]: I don't understand why we need lock here. Moreover, why should we restrict call of same sequence twice or call of different sequences concurrently if user wants to schedule them like that for whatever reason. Its users application design to deal with consequences of such scenarios if they build it like that in their task-plan/schedule.

**Review by bcwaldon — APPROVED** [review]
This looks to allow multiple sequences to execute in parallel, but not two instances of same sequence. If this is the intent, then LGTM.

I have not run this myself, to be clear.

**Pull request merged**